src/Codeception/Module/Apc.php

```diff
@@ -0,0 +1,235 @@
+            );
+        }
+
+        if (!ini_get('apc.enabled') || (PHP_SAPI === 'cli' && !ini_get('apc.enable_cli'))) {
+            throw new ModuleException(
+                __CLASS__,
+                'The "apc.enable_cli" parameter must be set to "On".'
+            );
+        }
+    }
+
+    /**
+     * Code to run after each test.
+     *
+     * @param TestInterface $test
+     */
+    public function _after(TestInterface $test)
+    {
+        $this->clear();
+    }
+
+    /**
+     * Grabs value from APC(u) by key.
+     *
+     * Example:
+     *
+     * ``` php
+     * <?php
+     * $users_count = $I->grabValueFromApc('users_count');
+     * ?>
+     * ```
+     *
+     * @param string|array $key
+     * @return mixed
+     */
+    public function grabValueFromApc($key)
+    {
+        $value = $this->fetch($key);
+        $this->debugSection('Value', $value);
+
+        return $value;
+    }
+
+    /**
+     * Checks item in APC(u) exists and the same as expected.
+     *
+     * Examples:
+     *
+     * ``` php
+     * <?php
+     * // With only one argument, only checks the key exists
+     * $I->seeInApc('users_count');
+     *
+     * // Checks a 'users_count' exists and has the value 200
+     * $I->seeInApc('users_count', 200);
+     * ?>
+     * ```
+     *
+     * @param $key
+     * @param $value
+     */
+    public function seeInApc($key, $value = null)
+    {
+        if (null === $value) {
+            $this->assertTrue($this->exists($key), "Cannot find key '$key' in APC(u).");
+        } else {
+            $actual = $this->grabValueFromApc($key);
+            $this->assertEquals($value, $actual, "Cannot find key '$key' in APC(u) with the provided value.");
+        }
+    }
+
+    /**
+     * Checks item in APC(u) doesn't exist or is the same as expected.
+     *
+     * Examples:
+     *
+     * ``` php
+     * <?php
+     * // With only one argument, only checks the key does not exist
+     * $I->dontSeeInApc('users_count');
+     *
+     * // Checks a 'users_count' exists does not exist or its value is not the one provided
+     * $I->dontSeeInApc('users_count', 200);
+     * ?>
+     * ```
+     *
+     * @param $key
+     * @param $value
+     */
+    public function dontSeeInApc($key, $value = null)
+    {
+        if (null === $value) {
+            $this->assertFalse($this->exists($key), "The key '$key' exists in APC(u).");
+        } else {
+            $actual = $this->grabValueFromApc($key);
+            if (false !== $actual) {
+                $this->assertEquals($value, $actual, "The key '$key' exists in APC(u) with the provided value.");
+            }
+        }
+    }
+
+    /**
+     * Stores an item `$value` with `$key` on the APC(u).
+     *
+     * @param string $key
+     * @param mixed $value
+     * @param int $expiration
+     */
+    public function haveInApc($key, $value, $expiration = null)
+    {
+        $this->assertTrue($this->store($key, $value, $expiration));
```

Comment:
there should not be assertions in `have*` methods. Just store it and return some key (if it is needed).
Also please provide an example in docblock